tokio/src/io/stderr.rs

```diff
@@ -67,8 +67,12 @@ cfg_io_std! {
     /// ```
     pub fn stderr() -> Stderr {
         let std = io::stderr();
+        // SAFETY: The `Read` implementation of `std` does not read from the
+        // buffer it is borrowing and correctly reports the length of the data
+        // written into the buffer.
+        let blocking = unsafe {Blocking::new(std)};
```

Comment:
Ditto in the other two files. Not sure why rustfmt didn't catch this.
```suggestion
        let blocking = unsafe { Blocking::new(std) };
```